css/templates/lune/_type.css

```diff
@@ -0,0 +1,31 @@
+@import 'page-vars';
+
+/* Bass CSS responsive typescale does not provide h0 and h00
+  below 640px screens. So we defined them here.
+  TODO: Raise issue with base css to get this fixed. */
+
+.h00 {
+  font-size: var(--h00);
+}
+
+.h0 {
+  font-size: var(--h0);
+}
+
```

Comment:
Yeah, check out the responsive type scale add on. Code here:
https://github.com/basscss/addons/blob/master/modules/responsive-type-scale

You'll notice inside `index.css`, `h0` and `h00` are defined. And inside `lib/sm-type-scale.css` the `.sm-h0` and `.sm-h00` classes are defined, but they sit within the `--breakpoint-sm` breakpoint, so we can't use these sizes on smaller devices.